src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -66,7 +66,7 @@ public DicomWebClient(HttpClient httpClient)
 
             request.Headers.TryAddWithoutValidation(
                 "Accept",
-                CreateAcceptHeader(DicomWebConstants.MediaTypeApplicationDicom, dicomTransferSyntax));
+                CreateAcceptHeader(CreateMultipartMediaTypeHeader(DicomWebConstants.ApplicationDicomMediaType), dicomTransferSyntax));
```

Comment:
Since we support both, we can leave the default as application/dicom in Client dll, what do you think?

Reply:
We also need to update the documentation on the new support. The conformance doc needs to be updated

Reply:
What understood from the issue conversation, we need to be backward compatible but going forward we need to support the new standard. Let me know if I understood that correctly. Good catch on the documentation. I will update it.

Reply:
Reverted to leave the client to accept application/dicom as default